frontend/www/js/omegaup/components/admin/Support.vue

```diff
@@ -4,6 +4,9 @@
       <div class="card-title h4">
         {{ T.omegaupTitleSupportDashboard }}
         <span v-if="username != null">- {{ username }} ({{ email }})</span>
+        <span v-if="contestAlias != null && contestTitle != null"
```

Comment:
Here you only need to check if `contestAlias` is not null. The rest can be removed. Even better, you can ensure that only one of the two texts (user or contest) is shown, so this can be changed to a `v-else-if`
```suggestion
        <span v-else-if="contestAlias != null"
```

Reply:
You are right, make me make the changes. Thank you!